Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.Members.cs

```diff
@@ -23,7 +23,7 @@ public partial class ImageExBase
         /// <summary>
         /// Identifies the <see cref="CornerRadius"/> dependency property.
         /// </summary>
-        public static readonly DependencyProperty CornerRadiusProperty = DependencyProperty.Register(nameof(CornerRadius), typeof(CornerRadius), typeof(ImageExBase), new PropertyMetadata(new CornerRadius(0)));
+        public static new readonly DependencyProperty CornerRadiusProperty = DependencyProperty.Register(nameof(CornerRadius), typeof(CornerRadius), typeof(ImageExBase), new PropertyMetadata(new CornerRadius(0)));
```

Comment:
Why is the `new` here?

Reply:
CornerRadius was added as a new property on `Control` starting 17763. We need the property down to 16299

Reply:
Will this be an issue/breaking change once 17763 becomes the min ver? It's not clear that we can remove this without breaking binary compat if that's something we're worried about. Just thinking ahead.

Reply:
Good question, but not sure if there is a better alternative?